### ISSUE TITLE
Making a note here re: OS X

### ISSUE THREAD

**Issue by hrbrmstr** [ua-parser/uap-r#9]
fails hard even with the C++ YAML lib installed. Will post logs in a bit. 

However…

Why not read the YAML file with R (via the `yaml` package) and pass an R data structure into Rcpp? It's not like the UA YAML file is huge (so it's not going to really save time) and it would remove something you'd no doubt have to make a download-and-install library as part of the pkg installation process (== ugh).

Also, an alternative to getting C++ regex working is to ask the `stringi` folks to expose a cpl functions directly (i.e. just use `stringi` regexes). I'll still keep poking at the example `icu` code you slack'd tho.

**Comment by Ironholds** [comment]
(Any luck getting the stringi example to work, btw?)

No big reason except that it's a pretty big refactor I haven't gotten around to, tbh.

**Comment by Ironholds** [comment]
@hrbrmstr so I'm now working on this and I've got everything working with one exception: C++11 uses a _different regex engine_ to Boost :|. If you find a PCRE to ECMAScript engine, or have any smart ideas, lemme know.

**Comment by bearloga** [comment]
FWIW, using the latest version of libyaml-cpp (via `brew install yaml-cpp`) and the latest version uaparser on macOS 10.12.5.

```R
uaparser::parse_agents("Mozilla/5.0 (Linux; Android 4.0.4; Galaxy Nexus Build/IMM76B) AppleWebKit/535.19 (KHTML, like Gecko) Chrome/18.0.1025.133 Mobile Safari/535.19")
```

Works fine (in Terminal, but not RStudio because #8)

**Comment by Ironholds** [comment]
Yeah, I'm REALLY looking forward to having the time to just rewrite all this shit
